### ISSUE TITLE
Question: Is Grails Console stealing logs?

### ISSUE THREAD

**Issue by kdybicz** [sheehan/grails-console#77]
If console is running on a working server, then wouldn't [this code](https://github.com/sheehan/grails-console/blob/master/grails3/plugin/grails-app/services/org/grails/plugins/console/ConsoleService.groovy#L21-L43) steal some of the System output (ie. logs from working jobs) from the output stream and never put them back?

**Comment by sheehan** [comment]
Yeah System.out is redirected to the web console while console code is executing. I was going to switch that to use TeeOutputStream or something like that, but never got to it.

**Comment by kdybicz** [comment]
I was thinking about doing it myself. I even forked this repo, but the code is too far gone for my skills... with the `fs` package gone I tried to update all dependencies and gulp script to do the job, but there seems to be a bit strange circular dependency sort off in this repo needed to build new package... and I gave up. Any chance you will be running any sort of maintenance of this repo anytime soon?

**Comment by kdybicz** [comment]
@sheehan if I would provide you a PR with updated node dependencies, gulp files converted to "latest standards" and some cleanup in `grails3/plugin` code, would you be keen to release new version of plugin?

FYI: my changes simply remove hijacking `System.out`, but I would be ok if you would like to make it configurable instead.

FYI 2: I tried my luck with TeeOutputSteam, but regardless the approach I was unable to get consistent results in case of concurrent code execution in 2+ console windows.

**Comment by kdybicz** [comment]
PR in question #78 😉
